utils/fix_cmocka_xml.py

```diff
@@ -32,7 +31,7 @@
         continue
     if file != "":
         print(file)
-        file_handle =  open('{0}'.format(file), "r+")
+        file_handle = open('{0}'.format(file), "r+")
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations